### PR TITLE
Add Accuracy Metric Card

### PR DESCRIPTION
**Pull request by emibaylor**
- adds accuracy metric card
- updates docstring in accuracy.py
- adds .json file with metric card and docstring information

### CODE REVIEW

File: metrics/accuracy/README.md [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+
+
+### Output Values
+- **accuracy**(`float` or `int`): Accuracy score. Minimum possible value is 0. Maximum possible value is 1.0, or the number of examples input, if `normalize` is set to `True`.. A higher score means higher accuracy.
+
+Output Example(s):
+```python
+{'accuracy': 1.0}
+```
+```python
```
Comment (sashavor) [inline metrics/accuracy/README.md:41]: Why is there a second empty code snippet?

**Review by sashavor — CHANGES_REQUESTED** [review]
A few minor issues (mostly adding some examples), otherwise this is great!

**Review by sashavor — APPROVED** [review]

File: metrics/accuracy/accuracy.py [hunk 2]
```diff
@@ -21,30 +21,47 @@
...
+        >>> accuracy_metric = datasets.load_metric("accuracy")
+        >>> results = accuracy_metric.compute(references=[0, 1, 2, 0, 1, 2], predictions=[0, 1, 1, 2, 1, 0])
+        >>> print(results)
+        {'accuracy': 0.5}
+
+    Example 2-The same as Example 1, except with `normalize` set to `False`.
+        >>> accuracy_metric = datasets.load_metric("accuracy")
+        >>> results = accuracy_metric.compute(references=[0, 1, 2, 0, 1, 2], predictions=[0, 1, 1, 2, 1, 0], normalize=False)
+        >>> print(results)
+        {'accuracy': 0.3}
```
Comment (lhoestq) [inline metrics/accuracy/accuracy.py:54]:
```suggestion
        {'accuracy': 3.0}
```

File: metrics/accuracy/accuracy.py [hunk 3]
```diff
@@ -21,30 +21,47 @@
...
-        Otherwise, return the fraction of correctly classified samples.
-    sample_weight: Sample weights.
+- **predictions** (`list` of `int`): Predicted labels.
+- **references** (`list` of `int`): Ground truth labels.
+- **normalize** (`boolean`): If set to False, returns the number of correctly classified samples. Otherwise, returns the fraction of correctly classified samples. Defaults to True.
+- **sample_weight** (`list` of `float`): Sample weights Defaults to None.
+
 Returns:
-    accuracy: Accuracy score.
+- **accuracy** (`float` or `int`): Accuracy score. Minimum possible value is 0. Maximum possible value is 1.0, or the number of examples input, if `normalize` is set to `True`.. A higher score means higher accuracy.
```
Comment (lhoestq) [inline metrics/accuracy/accuracy.py:40]:
```suggestion
    accuracy (`float` or `int`): Accuracy score. Minimum possible value is 0. Maximum possible value is 1.0, or the number of examples input, if `normalize` is set to `True`.. A higher score means higher accuracy.
```

**Review by lhoestq — APPROVED** [review]
I did a small change to fix the doctest with `normalize=False`, it should fix the CI.

I added some suggestions to keep the right format to the docstrings as well.

When the CI is green and the docstrings have the right format feel free to merge :)

**Pull request merged**